server/models/Expense.js

```diff
@@ -141,6 +142,32 @@ export default function (Sequelize, DataTypes) {
       deletedAt: {
         type: DataTypes.DATE,
       },
+
+      tags: {
+        type: DataTypes.ARRAY(DataTypes.STRING),
+        set(tags) {
+          if (tags) {
+            tags = tags
+              .map(tag => {
+                if (tag) {
+                  const upperCase = tag.toUpperCase();
+                  const cleanTag = upperCase.trim().replace(/\s+/g, ' ');
+                  return stripHTML(cleanTag);
```

Comment:
**[:sparkles: **Enhancement**]** As mentioned on Slack, this `stripHTML` is not required because we don't use tags in a `dangerouslySetInnerHtml`. I went a bit too far by adding it to conversations tags :slightly_smiling_face: